src/kiota/Handlers/BaseKiotaCommandHandler.cs

```diff
@@ -52,6 +61,15 @@ protected KiotaConfiguration Configuration
         configObject.BindConfiguration(configuration);
         return configObject;
     });
+
+    protected HttpClient GetHttpClient()
+    {
+        var httpClientHandler = new HttpClientHandler();
```

Comment:
both the http client and the handler should be added to the disposables collection for sanity